### DISCUSSION TITLE
致各位：一些交流方面的建议

### DISCUSSION THREAD

**Discussion by Pigeon0v0** [Meloong-Git/PCL#1592] [讨论]
近期反馈数量不断增加，也显示出了一些交流上的问题，下面我就个人看法发表一些建议：

（下文中 「协作者」 即 Collaborator，GitHub Mobile 在中文下是这么翻译的）

### 1. 在提交 Issue 之前 **一定一定** 要先检查是否有类似反馈，类似或重复的反馈会降低解决问题的效率。

如果你的问题和别人有重复，你可以前往对应的 Issue 下方留言，抑或是上传你的 日志 / 截图 / 复现步骤 。

你可以使用搜索功能快速搜索几个关键词来帮助确定。如果你实在不确定是否重复，也可以先提交，协作者会帮助你确定。

### 2. 使用正确的反馈分区。

在你提交 Issue / Discussion 之前，GitHub 会要求你选择一个分区。请确保你选择的分区类型正确，这有助于解决问题和提高效率。

### 3. 尽量不要在邮箱内直接回复。

别人看到你的回复时会附带上一串很长的文字，而这不利于查看内容。当你在邮箱内收到回复提示时，先前往 GitHub 网页端，或是 GitHub Mobile App，再进行回复。

### 4. 避免出现过长引用。

此类问题多出现在 Issues 下方，由于 Issues 没有 GitHub 提供的楼中楼，使用 Reply 回复功能会复制被回复消息原文。当你进行连续对话时，请确保不要出现 3 层以上的引用。这会降低阅读对话的效率。

### 5. 保证文件上传完成后再提交。

上传文件是一个过程，并不是你拽进来文件就马上上传好了的。如果你看见了 「Uploading」 字样，这就代表文件没有上传完成。请等待上传完成以后再提交相关内容。

那么我没上传完成就已经提交了怎么办？

### 6. 学会使用 Edit 编辑功能。

当你打错字，抑或是不小心在文件上传完成之前就提交了 Issue / Discussion / Comment ，请使用 GitHub 自带的 Edit 编辑功能来编辑你已经发送了的消息。

如果你是在收到他人提醒以后再补传的，请使用 Reply 回复功能提醒他，你已经补传好了。

### 7. 对于协作者：尽可能保证相关单词拼写正确。

单词拼写错误可能导致他人在使用翻译工具时遇到问题，抑或是看起来不是很舒服。

例如，标签是 Label 而不是 Lable 。

以上仅为个人看法，希望大家能共建一个更流畅的交流环境～

**Comment by WForst-Breeze** [comment]
支持~！希望可以和所有反馈者共同创造一个良好的社区讨论环境🥳

**Comment by iPodtouch6557** [comment]
更改：edit功能在右上角三个点-那条标识横线下面的edit按钮里
还有，可以把他推到龙猫那儿作为公告

**Reply by Axc077** [reply]
《edited》

**Comment by WorldHim** [comment]
如果你提交的 Bug 易于复现，建议在提交 Bug 前，重新打开 PCL，并再次复现，使 PCL 将复现过程在新 `Log1.txt` 中，以便查看日志。~~（动辄几千行的日志里面找 Bug 真的害人不浅啊~~

**Reply by MoYuan-CN** [reply]
或者贴一下大概在第几行

**Comment by Louis-Harsune** [comment]
天哪，把Label打成Lable对我来说是真的高血压（233

**Reply by iPodtouch6557** [reply]
这其实怪不得人，因为如果你到百度去搜，会发现既有认为是lable也有label的，而且一段时间前我用的搜狗输入法两个都打的出来（现在只能打label了），并且**往往这些人甚至都不会关注github是咋拼的，就瞎打一通**
:.(

**Comment by 20030229** [comment]
能不能加个存档的fabric的更新啊？旧存档需要用的是0.14.10版本的fabric，但是模组需要更高版本，我手动去更新fabric好麻烦，官网下载器什么的我都试了，还是更新不了旧存档的fabric，能不能像hmcl那样能在启动器里更新fabric呢，这对于我这种小白很重要啊的操作

**Reply by Chiloven945** [reply]
你所回复的内容与这个主题没有任何关系，请自己单独开一个
也不要在每一个地方都回复一遍，这样很招人厌
